xdsl/xdsl_opt_main.py

```diff
@@ -112,6 +113,7 @@ def get_all_passes() -> list[type[ModulePass]]:
         LowerSnitchRuntimePass,
         PrintToPrintf,
         RISCVRegisterAllocation,
+        RISCVLowerArithRV32,
```

Comment:
Wondering whether we at some point should integrate the ABI spec thing from @AntonLydike here.
Not in this PR?

Reply:
Some part of the lowering is really specific and assume a 32 bits target. I just wanted to make it clear.

Reply:
@superlopuh does your comment about the pass name also apply here?